### DISCUSSION TITLE
Modifier Keys as single key shortcuts

### DISCUSSION THREAD

**Discussion by isaacplmann** [johnlindquist/kit#758] [Ideas]
I would like to assign a shortcut to a script of just the left command key or just the right option key.

i.e.
```
// Shortcut: left_command
```

I currently set this up in Karabiner Elements, but the syntax is tedious and I'd like to move all my script management into Script Kit.

**Comment by johnlindquist** [comment]
Yeah, I'd love to be able to provide that. The only way I can think to solve that (for global shortcuts) would be to bundle karabiner with the app and write some wrappers around it. It's definitely on my "someday" list.

Re: tedious - Are you using https://github.com/yqrashawn/GokuRakuJoudo with Karabiner? It makes it _way_ better than managing a huge json file. Still far from ideal though.

**Comment by johnlindquist** [comment]
Hmm, I may have found an alternate solution. Investigating! 🧐
